monitor/redis/watcher.go

```diff
@@ -70,20 +70,35 @@ func (w *Watcher) monitor(ctx context.Context, ch chan<- []*change.Change) {
 	}
 }
 
+func toPtr(s string) *string {
```

Comment:
can we inline this? it is only used in one place.

Reply:
done